pkg/valueset/postgresql_driver.go

```diff
@@ -0,0 +1,58 @@
+package valueset
+
+import (
+	"fmt"
+
+	"github.com/jackc/pgx/v5"
+	_ "github.com/jackc/pgx/v5/stdlib"
+)
+
+// match (n:OMOPConcept) return n
+func GetResults(cfg Config, queryParams map[string]string) (interface{}, error) {
+	ret := make([]map[string]interface{}, 0)
+DATABASE:
+	for _, dbs := range cfg.Databases {
+		for _, vs := range dbs.Valuesets {
+			if err := cfg.GetConnection(vs, &dbs.Database); err != nil {
+				return nil, err
+			}
+			if dbs.Driver == nil {
+				continue DATABASE
+			}
+			for _, query := range vs.Queries {
```

Comment:
Find the queries in vs using tableId
Then go through each query using the queryparams map[string]string
Do not hadrcode any key other than tableId